src/app-layer-htp.c

```diff
@@ -560,6 +562,7 @@ struct {
     { "Request buffer over", HTTP_DECODER_EVENT_REQUEST_FIELD_TOO_LONG},
     { "Response buffer over", HTTP_DECODER_EVENT_RESPONSE_FIELD_TOO_LONG},
     { "C-T multipart/byteranges in responses not supported", HTTP_DECODER_EVENT_RESPONSE_MULTIPART_BYTERANGES},
+    { "Compression bomb:", HTTP_DECODER_EVENT_COMPRESSION_BOMB},
```

Comment:
This comes from libhtp : https://github.com/OISF/libhtp/blob/0.5.x/htp/htp_transaction.c#L795

That can work without `:` as well
I tried to be more complete, so if a new log message such as `Compression bombing` comes, it will not trigger...

So, what do you think ?

Reply:
I think we can leave as-is.